### PR TITLE
Issue 81 More issues fixed

### PR DESCRIPTION
**Pull request by alrusdi**
Hooray! Just 9 issues remaining. We are close to finish this huge task.
Please review, merge, but do not remove branch

### CODE REVIEW

File: src/cards/Fish.ts [hunk 1]
```diff
@@ -15,19 +15,40 @@ export class Fish implements IActionCard, IProjectCard {
   public name: string = 'Fish';
   public resourceType: ResourceType = ResourceType.ANIMAL;
   public cardType: CardType = CardType.ACTIVE;
+
+  private getPlayersWithPlantProduction(currentPlayer: Player, game: Game): Array<Player> {
+    var players = game.getPlayers().filter((p) => p.plantProduction > 0);
+    if (players.length === 2) {
```
Comment (bafolts) [inline src/cards/Fish.ts]: Why does this have to equal 2?

File: src/cards/HeatTrappers.ts [hunk 2]
```diff
@@ -11,20 +11,46 @@ export class HeatTrappers implements IProjectCard {
     public tags: Array<Tags> = [Tags.ENERGY, Tags.STEEL];
     public name: string = "Heat Trappers";
     public cardType: CardType = CardType.AUTOMATED;
-    public canPlay(): boolean {
-        return true;
+
+    private getPlayersWithHeatProduction(currentPlayer: Player, game: Game): Array<Player> {
+        var players = game.getPlayers().filter((p) => p.heatProduction > 1);
+        if (players.length === 2) {
```
Comment (bafolts) [inline src/cards/HeatTrappers.ts]: This is becoming a pattern. I am not sure what is special about 2 players. I would think this should be `if (players.length > 1)`

Comment (alrusdi) [reply src/cards/HeatTrappers.ts]: You are right! Wrong decision and then copy paste.

**Comment by alrusdi** [comment]
I resolved review issues and added fixes for remaining cards.
I everything is alright please merge and delete the branch. We still have some problems (with Protected Habitatas and Pets) but they need a sort of common solution. I will create new issue for it and ask @bafolts for help

**Pull request merged**